Doc/library/pathlib.rst

```diff
@@ -212,7 +212,10 @@ Paths of a different flavour compare unequal and cannot be ordered::
 Operators
 ^^^^^^^^^
 
-The slash operator helps create child paths, similarly to :func:`os.path.join`::
+The slash operator helps create child paths, mimicking the behaviour of
+:func:`os.path.join`. For instance, when several absolute paths are given, the
+last is taken as an anchor; for a Windows path, changing the local root doesn't
+discard the previous drive setting::
```

Comment:
```suggestion
The slash operator helps create child paths, like :func:`os.path.join`. 
If the argument is an absolute path, all previous path components are thrown away.
On Windows, the drive letter is not reset when a rooted driveless component 
(e.g., `r'\foo'`) is encountered::
```

We actually use `os.path.join()` under the hood now, so 'mikicking the behaviour' is perhaps misleading. Suggest simplifying to "like os.path.join()"

I don't much like "taken as an anchor" because the last absolute path component can contribute _more_ than just the anchor (where I'm defining "anchor" per pathlib). I also don't like 'changing the local root', as the root _doesn't_ change in the example given.